src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -29,16 +33,19 @@
 
     // Whether to do debug output (i.e. leave output around).
     // This should always be false when checked in.
-    private static final boolean doDebugTests           = false;
-    private static final String  LARGE_DATASOURCES_FOLDER = "funcotator_dataSources_latest";
+    private static final boolean doDebugTests = false;
+    private static final String LARGE_DATASOURCES_FOLDER = "funcotator_dataSources_latest";
+
+    private static final String PIK3CA_VCF_HG19 = toolsTestDir + "funcotator/0816201804HC0_R01C01.pik3ca.vcf";
```

Comment:
I'm not sure this file is publicly releasable - I believe it needs to be sanitized before inclusion in our tests.

Reply:
It is publicly releasable...

No action.